### PR TITLE
Fix pep8

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

File: hr_expense_accountedge/tools/import_expenses.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def main():
     sock_common = xmlrpclib.ServerProxy('%s/xmlrpc/common' % server_url)
     uid = sock_common.login(dbname, username, pwd)
     if not uid:
-        print "Connection error. Please check the username, password and server url."
+        print("Connection error. Please check the username, password and server url.")
```
Comment (pedrobaeza) [inline hr_expense_accountedge/tools/import_expenses.py:39]: Maybe not in the scope of this PR, but better use logger.

File: hr_expense_accountedge/tools/import_expenses.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def main():
     # Search for exported expense notes
     args = [('state', '=', 'exported')]
     expense_ids = sock.execute(dbname, uid, pwd, 'hr.expense.expense', 'search', args)
-    print "There are %d expense sheets to import" % len(expense_ids)
+    print("There are %d expense sheets to import" % len(expense_ids))
```
Comment (pedrobaeza) [inline hr_expense_accountedge/tools/import_expenses.py:47]: Maybe not in the scope of this PR, but better use logger.

**Comment by pedrobaeza** [comment]
:+1: (code review)

Two comments out of the scope of PEP8

**Comment by rvalyi** [comment]
:+1: 

BTW, hey guys, we are busy with other stuff right now, but don't worry, we are coming back on all these OCA PR's; kudos for the work meanwhile!

File: hr_expense_accountedge/tools/import_expenses.py (hunk 1 above)
Comment (bwrsandman) [inline hr_expense_accountedge/tools/import_expenses.py:39]: Can you make an issue mentioning those two files?

**Pull request merged**